build-and-push-spring-boot-application/action.yaml

```diff
@@ -30,16 +30,19 @@ runs:
       with:
         distribution: temurin
         java-version: ${{ inputs.java-version }}
-        cache: 'gradle'
     - run: echo '::set-output name=IMAGE_NAME::gcr.io/${{ inputs.project-id }}/${{ inputs.deployment-name }}'
       shell: bash
       name: Generate the name of the docker image
       id: generate-image-name
     - uses: gradle/gradle-build-action@v2
-      name: Install Gradle Wrapper
-    - run: ./gradlew clean bootBuildImage -Partifactory_user=${{ inputs.artifactory-username }} -Partifactory_password=${{ inputs.artifactory-password }} --imageName=${{steps.generate-image-name.outputs.IMAGE_NAME}}:${{ github.sha }}
-      shell: bash
       name: Run Gnag and build Docker Container
```

Comment:
🙈 I think I forgot to remove this. We might want to update the name to `Build Docker Container`

Reply:
@Janani-Subbiah good catch, it should be updated now